### ISSUE TITLE
For reference time series it is not possible to add users to access control.  The interface element is missing

### ISSUE THREAD

**Issue by dtarb** [hydroshare/hydroshare#329]
![capture](https://cloud.githubusercontent.com/assets/3456833/7288600/379cc564-e932-11e4-8483-fa9997d0203f.PNG)

**Comment by dtarb** [comment]
@jsadler2 is this still a problem.

**Comment by dtarb** [comment]
@danames please look at this issue and decide what needs to be done.

**Comment by zhiyuli** [comment]
@dtarb  We are having some problems on merging refts with new base model and new access control. Although this issue is related to old access control, we leave it for now and will come back to check it as soon as big merge is done.

**Comment by dtarb** [comment]
@zhiyuli the big merge has already been done as far as the develop branch is concerned.  There is work to tranfer resources from the old system to the updated system.  But there is no need for you to wait.  You wan work right now on this issue using the develop branch code base.

**Comment by zhiyuli** [comment]
@dtarb  
Yes, we have been working on merging RefTS and Geographic Feature with latest dev since last Monday. We are having some problems and Pabitra and Calvin are helping us. 

We chose to start from the geographic feature res type because its workflow is similar to other res types but we got issue 636 (more details see team mail list and hipchat).  Pabitra and I did a step by step debugging and found out this is issue may be related to the mixing of Mazzine access control and new access control. Pabitra will talk with Calvin about this. In the meantime, I have moved on to working on merging RefTS with last dev and got new problems.
